bespoke.xcodeproj/project.pbxproj

```diff
@@ -343,35 +392,14 @@
 			buildActionMask = 2147483647;
 			files = (
 				9F17849E2A984B58009F2549 /* .swiftlint.yml in Resources */,
+				9F372F792AC216E300F45D47 /* GoogleService-Info.plist in Resources */,
 				9F17845F2A93C677009F2549 /* .gitignore in Resources */,
 				9F1784A92A9854D5009F2549 /* .swiftformat in Resources */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
 /* End PBXResourcesBuildPhase section */
 
-/* Begin PBXShellScriptBuildPhase section */
-		9F17849F2A984C1B009F2549 /* Run Script */ = {
-			isa = PBXShellScriptBuildPhase;
-			alwaysOutOfDate = 1;
-			buildActionMask = 2147483647;
-			files = (
-			);
-			inputFileListPaths = (
-			);
-			inputPaths = (
-			);
-			name = "Run Script";
-			outputFileListPaths = (
-			);
-			outputPaths = (
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "if [[ \"$(uname -m)\" == arm64 ]]; then\n    export PATH=\"/opt/homebrew/bin:$PATH\"\nfi\n\nif which swiftlint >/dev/null; then\n    swiftlint --fix && swiftlint\nelse\n    echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
-		};
-/* End PBXShellScriptBuildPhase section */
```

Comment:
I deleted SwiftLint cause it was causing builds to fail (maybe since I updated Xcode, not sure) - can add it back but also whatever(?)